highfidelity/hifi
# FB11123 Make the certificate GOLD (and change title to Proof of Prove…

…nance License)

https://highfidelity.fogbugz.com/f/cases/11123/Make-the-certificate-GOLD-and-change-title-to-Proof-of-Provenance-License
  
***Test plan***

1. Go to market
2. Buy some item & rezz (I used box of balls)
3. Inspect it by pointing half-pulled laser to item
4. Ensure certificate info has golden background and font colors like on the following sample: 

![image](https://user-images.githubusercontent.com/23638/34684724-18496ab2-f4b7-11e7-82fb-f23cc6881e42.png)

5. Ensure title changed from 'PROOF OF PURCHASE' to 'Proof of Provenance'
